homework03/arrays/arrays.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import "fmt"
+
+func main() {
+    nums := []int{1, 2, 3, 4, 5, 6}
+    average(nums...)
+}
+
+func average(nums ...int) {
```

Comment:
What's the use of using variadic function parameters? It won't work with arrays anyway